app/state_machines/verification_attempt_state_machine.rb

```diff
@@ -11,15 +11,15 @@ class VerificationAttemptStateMachine
   transition from: :escalated, to: [:approved, :denied, :requested_replacements]
 
   after_transition(to: :approved, after_commit: true) do |verification_attempt, transition|
-    verification_attempt.client.update(identity_verified_at: transition.created_at)
+    verification_attempt.client.update(identity_verified_at: transition.created_at, identity_verification_denied_at: nil)
     efile_submission = verification_attempt.client.efile_submissions&.last
     if efile_submission.present? && efile_submission.in_state?(:fraud_hold)
       efile_submission.transition_to(:resubmitted, { initiated_by_id: transition.metadata[:initiated_by_id] })
     end
   end
 
   after_transition(to: :denied, after_commit: true) do |verification_attempt, transition|
-    verification_attempt.client.update(identity_verification_denied_at: transition.created_at)
+    verification_attempt.client.update(identity_verification_denied_at: transition.created_at, identity_verified_at: nil)
```

Comment:
Same comment as above. Since the variable is nil anyway why do even have to define it?

Reply:
this is wiping out a value if it had been previously set